### ISSUE TITLE
postPackageUpdateAndInstall not run on multiple package install

### ISSUE THREAD

**Issue by marcrobertscamao** [neos/flow-development-collection#2620]
### Description

Upon installing a project either through create-project or when checking out a new project and running composer install the postPackageUpdateAndInstall is not run for all of the installed packages.

### Steps to Reproduce

1. check out a new neos package with multiple uses of installer-resource-folders
2. composer install

#### Expected behavior

I should see all of the resources defined by the installer-resource-folders available in the installed application

#### Actual behavior

Only one of the packages wins the race to postPackageUpdateAndInstall and will have resources installed, all others are ignored

### Affected Versions

Neos: 7.2.0

Flow: 7.2.1

**Comment by sorenmalling** [comment]
Hi @marcrobertscamao 

Can you provide a link to a repository with a Flow package, where the files are not being copied as expected?

I know for sure that the carbon/pipelin package has the lines

```
    "extra": {
        "neos": {
            "installer-resource-folders": [
                "/Installer/"
            ]
        }
    }
```
in the composer.json and files are copied correctly - so will love to see a example confirming the issue

**Comment by marcrobertscamao** [comment]
Hi @sorenmalling,

To reproduce this you can do the following:

```bash
git clone git@github.com:marcrobertscamao/resource-test.git
cd resource-test
composer install
```

Here you should see `resource1.txt` and `resource2.txt` in the at `./`

If you then remove the packages and repeat then you will see the problem:

```bash
rm -rf Packages/Application/Installer.*
composer install
```

After this you will see resource1.txt is installed and can then rinse and repeat:

```bash
rm -rf Packages/Application/Installer.Resource2
composer install
```

Now you see both resource files.

PS. I have added a pull request to fix: https://github.com/neos/flow-development-collection/pull/2621

Regards,
Marc

**Comment by sorenmalling** [comment]
Closed by #2621
